src/ray/util/signal_handler.cc

```diff
@@ -0,0 +1,72 @@
+#include "ray/util/signal_handler.h"
+#include "ray/util/logging.h"
```

Comment:
I think this import should come later according to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes